.github/workflows/e2e-fixture-test.yml

```diff
@@ -0,0 +1,34 @@
+name: E2E Fixture Test
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
```

Comment:
We should also add:
- SECURITY_CONTACTS
- DCO
- OWNERS

So it might be easier to add a matching rule for files that are all caps and have no suffix, unless changes to `PROJECT` has effect on e2e.

It would also be nice to ignore `hack/*.sh`, since they're for Prow/local.